system_test/common/aest_commands_SUITE.erl

```diff
@@ -92,6 +92,17 @@ commands_(OpsBin, Cfg) ->
     {0, _Output8} = run_cmd_in_node_dir(node1, [OpsBin, "export", GuestPath], Cfg),
     ?assert(filelib:is_regular(HostPath)),
 
+    {0, Output9} = run_cmd_in_node_dir(node1, [OpsBin, "rename_db", "./data/mnesia/schema.DAT"], Cfg),
```

Comment:
So this is run while node running? It means that any lines added afterwards in this test case would find changed case. I suggest deleting or moving to separate test case.